File: Kattis/Python/Left-Beehind.py
def main():
    
    x, y = input().split()
    x, y = int(x), int(y)

    if x == 0 and y == 0:
        return 0 

    while True:

        if x + y == 13:
            print("Never speak again.")
        elif x > y:
            print("To the convention.")
        elif x < y:
            print("Left beehind.")
        else:
            print("Undecided.")
        
        x, y = input().split()
        x, y = int(x), int(y)
        
        if x == 0 and y == 0:
            break
    

if __name__ == '__main__':
    main()

File: Kattis/C++/Left-Beehind/Left-Beehind.cpp
#include <bits/stdc++.h>

using namespace std;
using i64 = long long;

int main()
{
    i64 x,y;
    cin >> x >> y;
    
    while(x != 0 || y != 0)
    {
        if (x + y == 13)
        {
            cout << "Never speak again." << endl;
        }
        else if(x > y)
        {
            cout << "To the convention." << endl;
        }
        else if(x == y)
        {
            cout << "Undecided." << endl;
        }
        else
        {
            cout << "Left beehind." << endl;
        }
        cin >> x >> y;
    }
    return 0;
}
